File: 1ºAno/Lista2/C/Sala/files/ex2.c
#include <stdio.h>

int main() {

    float n1, n2, e;
    printf ("Entre com as notas, vossa senhoria:\n");
    printf ("N1 = ");
    scanf ("%f", &n1);
    printf ("N2 = ");
    scanf ("%f", &n2);
    printf ("Média = %.2f\n", (n1 + n2) / 2);
    if ((n1 + n2) / 2 >= 6) {
        printf ("Aluno aprovado :)\n");
    }
    else {
        printf ("Infelizmente, precisaremos da nota de exame:\n");
        printf ("E = ");
        scanf ("%f", &e);
        printf ("Nova média = %.2f\n", (n1 + n2 + e) / 3);
        if ((n1 + n2 + e) / 3 >= 6) {
            printf ("Aluno aprovado em exame :)\n");
        }
        else {
            printf ("Aluno reprovado :(\n");
        }
    }

    return 0;
}

File: 1ºAno/Lista2/Python/Sala/files/ex2.py
n1 = float(input("Entre com as notas:\nN1 = "))
n2 = float(input("N2 = "))
if (n1 + n2) / 2 >= 6:
    print ("Aprovado :)\nMédia = %.2f" % ((n1 + n2) / 2))
else:
    e = float(input(("Eh rapaz, passa aí a nota do exame: ")))
    print ("Aprovado :)") if (n1 + n2 + e) / 3 >= 5 else print ("Reprovado :(") 
    print ("Média = %.2f" % ((n1 + n2 + e) / 3))